src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -964,8 +965,21 @@ private void visitProperties(ObjectType objType, boolean isStatic, Set<String> s
             || "constructor".equals(propName)) {
           continue;
         }
+        if (isPrivate(objType.getOwnPropertyJSDocInfo(propName))) {
+          hasPrivateProps = true;
+          continue;
```

Comment:
It would be safe to continue into the `visitProperty` call, which already has private property check. That way if we need to modify whether we emit private properties we have to check any only one place. `visitProperties` is only concerned with emitting the no*Typing props.